lib/file-io/index.js

```diff
@@ -1,7 +1,10 @@
 const glob = require("glob");
+const fs = require("fs");
 
-function open(pattern){
-    return new Promise((resolve, reject) => {
+const encoding = "utf8";
+
+function open(pattern) {
```

Comment:
Should this function be called _find_ since it only returns an array of filenames?